src/lib/helpers/getNextDistDir.js

```diff
@@ -1,9 +1,11 @@
 // Get the NextJS distDir specified in next.config.js
 const { join } = require('path')
+
 const getNextConfig = require('../../../helpers/getNextConfig')
+const { defaultFailBuild } = require('../../../helpers/defaultFailBuild')
 
-const getNextDistDir = async () => {
-  const nextConfig = await getNextConfig()
+const getNextDistDir = async (publishPath) => {
+  const nextConfig = await getNextConfig(defaultFailBuild, publishPath)
```

Comment:
```suggestion
  const nextConfig = await getNextConfig({ publishPath })
```

ya like mickael said if you're defaulting failBuild in getNextConfig as an obj param then youd only have to pass publishPath here :)